src/logic/waila.ts

```diff
@@ -49,6 +50,7 @@ export function toParsedParams(
         ok: true,
         value: {
             address: params.address,
+            pj_uri: params.pj_uri,
```

Comment:
instead of doing this how I think we are gonna handle it is have it called `payjoin_enabled` and we set that as `params.payjoin_endpoint !== undefined`

then when we call `send_payjoin` we use the original input string. that way we can just let pdk handle it

Reply:
I just published waila v0.2.3 that has this function